torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -846,6 +846,15 @@ const std::vector<std::string> functions = {
 
             return result, backward
 
+        def hardswish(self):
+            result = torch.relu(self)
+            def backward(grad_output):
+                m = (result >= 3.).type_as(result)
+                m = torch.where((result >= -3.) & (result <= 3.),  result / 3. + .5, m)
```

Comment:
This is neat using `relu(self)>3.` to compute the derivative of hardswish when self is out of the range of (-3,3).

But shouldnt the condition be `(result > -3.) & (result < 3.)`?

Reply:
ooh i think i must have bad copy-pasta'd in the definition of hardswish (thus the test failures). should be `result = torch.hardswish(relu)`. 

Looks like it should be one >= and one < judging by the activations file https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/cpu/Activation.cpp#L466

Reply:
Okay so actually, it doesn't matter at x == 3, since if `x = 3` then `x * (x + 3)/6 == x`. It does matter when x = -3, and that's why we have the result >= -3. condition. However it would be clearer if I rewrote `m = (result >= 3.).type_as(result)` to be `m = (result >= 3.).type_as(result)`.

Reply:
Interesting, if we use the formula of `hardswish(x) = x` when x=3, the derivative is 1; but if we use `hardwish(x)= x(x+3)/6`, the derivative is `x/3+.5` (which is 1.5). I guess derivative at the boundary point is not precise.